tests/resources/sample_config.yaml

```diff
@@ -21,8 +21,10 @@ grains:
   threshold_method: otsu # Options : otsu, std_dev, absolute
   otsu_threshold_multiplier: 1.7
   threshold_std_dev: 1.0
-  threshold_absolute_lower: -1.0
+  threshold_absolute_lower: 1.0
   threshold_absolute_upper: 1.0
+  upper_absolute_area_threshold: [400,600] # above surface [Low, High] in nm^2 (also takes null)
```

Comment:
The other two configs have `[500, 800]` just wanted to check that these are meant to be as they are, I'd expect the tests you've written work with these.

Reply:
So yep this took me a while to figure out. But basically the differences are due to the grain finding method in this file being "otsu" which with the default values makes the grains a little skinnier and thus reduces their area compared to the other files which use the standard deviation method.

Reply:
Great, thanks for the clarification.

It sounds as though having sensible "`default`" configuration is going to be challenging!